### DISCUSSION TITLE
I have a problem setting up KENS

### DISCUSSION THREAD

**Discussion by jsjclink** [ANLAB-KAIST/KENSv3#27] [Project Q&A 1. Application Layer (Socket)] [answered]
Hi TA, I have a problem setting up KENS.

I used WSL2 and ubuntu 20.04 CLI in my local computer to set KENS.

https://github.com/ANLAB-KAIST/KENSv3/wiki/Getting-Started%3A-CLI-%28Linux%29
I set KENS by following the instructions in the above link.
And when I typed cmake -B build . I found following warnings..

Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
Looking for pthread_create in pthreads - not found
Could Not find Doxygen(missing: DOXYGEN_EXECUTABLE)
![image](https://user-images.githubusercontent.com/60886172/158804756-442df073-d7a7-4a71-bab3-b00847e4ff39.png)

I'm wondering if I can ignore that warnings like question #22.

Thank you.

**Comment by Pusnow** [comment, marked as answer]
You can dismiss the warnings.

**Reply by jsjclink** [reply]
Thank you!

**Comment by Jongjunp** [comment]
In my case, I installed 'doxygen' to remove that message.
In macOS, `brew install doxygen`

**Reply by jsjclink** [reply]
Thank you!!
